pkg/traceql/engine.go

```diff
@@ -205,7 +209,11 @@ func (s Static) asAnyValue() *common_v1.AnyValue {
 				StringValue: "nil",
 			},
 		}
-	default:
-		panic(fmt.Errorf("static has unexpected type %v", s.Type))
+	}
+
+	return &common_v1.AnyValue{
+		Value: &common_v1.AnyValue_StringValue{
+			StringValue: fmt.Sprintf("error formatting val: static has unexpected type %v", s.Type),
```

Comment:
I think this will make the error message show up in the results table, which is probably ok.